### PR TITLE
Competitor docs

### LINKED ISSUE #324: Add comparison to LGTM.co
I would like to see a comparison to https://lgtm.co/docs/overview/ in the wiki. I'm sure zappr already offers more features.

### PR DESCRIPTION
**Pull request by prayerslayer**
Fixes #324

### CODE REVIEW

**Comment by prayerslayer** [comment]
cc @LappleApple @mfellner

File: docs/competitors.md [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
 
-[PullApprove](https://pullapprove.com) works also technically the same like Zappr. Noteworthy:
+[PullApprove](https://pullapprove.com) works, from a technical perspective, very much like Zappr does. Noteworthy differences:
 
-* PullApprove supports private repositores, currently we do not. We will support them in the future.
-* Some of PullApprove's features [cost money](https://pullapprove.com/pricing/), Zappr is 100 % free as in beer and freedom.
-* PullApprove has a custom web UI you can use to give approvals. Zappr strives to offer only the minimal necessary web UI as you should do your work on GitHub to avoid friction in your workflow.
-* As of now every customization available in PullApprove is also available in Zappr.
+* PullApprove supports private repositories, whereas currently we do not. We will support them in the future, though.
+* Some of PullApprove's features [cost money](https://pullapprove.com/pricing/). As stated above, Zappr is free.
```
Comment (prayerslayer) [inline docs/competitors.md]: I liked beer and freedom in there :)

**Comment by prayerslayer** [comment]
👍

File: docs/competitors.md (hunk 1 above)
Comment (LappleApple) [inline docs/competitors.md]: But "think of the children!"

Comment (prayerslayer) [inline docs/competitors.md]: It's also a popular comparison when talking about OSS, to clarify whether you mean free as in "no money" or free as in "do what you want".

**Comment by taojang** [comment]
:+1:

**Pull request merged**